### ISSUE TITLE
Help!! Not Individually Addressable

### ISSUE THREAD

**Issue by dapiii** [jasoncoon/esp8266-fastled-webserver#115]
Doesnt matter what I do I cant manage to have just a single color to be displayed, for example when I set it to Solid Color and then Blue it just makes a Red Green and Blue pattern every 3 leds. Also if I set the number of LEDS to 240 which is the correct number for my strip (4m - 60led/m) then about 60 leds dont light up.  Im using a SK6812 RGBW strip which should have been working fine as it is very similar to the WS2812B but with an addition of a white led.

**Comment by atatistcheff** [comment]
Have you tried a different sketch with your SK6812 strip?  As of last year FastLED still didn't support the RGBW pixels although I read somewhere that it does now for the ESP8266.

**Comment by dapiii** [comment]
rgbw is still not properly supported but its being worked on. I have tried most of the FastLED examples and none seem to work at all. It appears the first LED in the strip is the only one working properly, as its always displaying the right color in whatever I tell it to do, but its a different story for the rest of the strip: it doesnt matter what animation/solid color its always showing a red green blue pattern, with slight hue variations depending on the animation, so rn its basically a christmas led strip except for the first led  . I have also tried Neopixel but nothing happens when I try an example.
